packages/scheduler/src/Scheduler.js

```diff
@@ -336,6 +347,31 @@ function unstable_scheduleCallback(callback, deprecated_options) {
   return newNode;
 }
 
+function unstable_pauseExecution() {
+  isSchedulerPaused = true;
```

Comment:
Why isn't the browser's built-in pause/resume functionality sufficient for whatever you're using this for?